test/controllers/notices-setup.controller.test.js

```diff
@@ -817,7 +817,7 @@ describe('Notices Setup controller', () => {
       beforeEach(() => {
         getOptions = {
           method: 'GET',
-          url: basePath + `/${session.id}/preview-return-forms`,
+          url: basePath + `/${session.id}/preview-return-forms/1234`,
```

Comment:
I'd still use something more representative, in this case an actual UUID.

```suggestion
          url: basePath + `/${session.id}/preview-return-forms/95b54f97-fefb-46e7-aae8-ebf40ecb8b50`,
```